source/include/TypeAndDataManager.h

```diff
@@ -1111,18 +1110,16 @@ class TypedArrayCore
     AQBlock1* BeginAtND(Int32, IntIndex*);
     AQBlock1* BeginAtNDIndirect(Int32 rank, IntIndex* ppDimIndexes[]);
 
- public:
-    void* RawObj()                  { VIREO_ASSERT(Rank() == 0); return RawBegin(); }  // some extra asserts fo  ZDAs
-    AQBlock1* RawBegin()            { return _pRawBufferBegin; }
+    void* RawObj() { VIREO_ASSERT(Rank() == 0); return RawBegin(); }  // some extra asserts fo  ZDAs
+    AQBlock1* RawBegin() { return _pRawBufferBegin; }
     template<typename CT> CT BeginAtAQ(IntIndex index) { return reinterpret_cast<CT>(RawBegin() + index); }
     BlockItr RawItr()               { return BlockItr(RawBegin(), ElementType()->TopAQSize(), Length()); }
 
- public:
     //! Array's type.
-    TypeRef Type()                  { return _typeRef; }
+    TypeRef Type() { return _typeRef; }
```

Comment:
It's ok here and no need to address this.. In the future, ,it is a good idea to do format/style changes (such as whitespace, newline) separately in a PR.